### PR TITLE
Add better buttons

### PR DESCRIPTION
**Pull request by superdav42**
For #13 
I think it's an improvement 
image

### CODE REVIEW

File: src/components/Buttons.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useMemo } from "react";
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+import {
+  ViewStream,
+  Subject,
+  Edit,
+  Preview,
+  Undo,
+  Redo, Save
```
Comment (mandolyte) [inline src/components/Buttons.jsx:9]: @superdav42 could add tool tips on these. Some are more international than others. At some point, we should consider some localizations on the tool tips, but that can wait.

Comment (mandolyte) [inline src/components/Buttons.jsx:9]: The save button does not indicate when there are saves pending. Could be probably be tied to the length of the history?

**Review by mandolyte — CHANGES_REQUESTED** [review]
You are right - these buttons look **much** nicer. Two things in the comments:
- provide tool tips
- make the state change of the save button reflect whether any changes have not been saved. I think this might easily be done by using the length of the undo history stack?

**Comment by superdav42** [comment]
> make the state change of the save button reflect whether any changes have not been saved. I think this might easily be done by using the length of the undo history stack?

Yep it was pretty easy. Please check it out.

**Pull request merged**